words/teams.txt

```diff
@@ -1,27 +1,136 @@
+aerie
 alliance
 assembly
+bale
 band
+barrel
 batch
+bed
+bevy
+board
+body
+brood
+building
 bunch
+business
+cackle
+camp
+cast
+catch
+cauldron
+charm
+chattering
+chime
+choir
+circle
+clan
+class
+clattering
+cloud
+clowder
 club
+cluster
+coalition
+colony
 combination
+committee
+company
 conglomerate
+congregation
+congress
+conspiracy
+convocation
+corporation
 coven
 crew
+culture
+dazzle
+descent
+doctrine
+dole
+drift
+drove
+exaltation
 faction
+faculty
+family
+flight
+fling
+flock
+flush
+gaggle
+galaxy
+gam
+game
 gathering
+gobble
 group
+gulp
+hang
```

Comment:
I would recommend against this because in combo with other words it could be inappropriate like `hang-people`